SYCL/Basic/enqueue_barrier.cpp

```diff
@@ -5,7 +5,7 @@
 
 // The test is failing sporadically on Windows OpenCL RTs
 // Disabling on windows until fixed
-// UNSUPPORTED: windows
+// UNSUPPORTED: hip_amd, windows
```

Comment:
There's a PR to fix these for HIP:

* https://github.com/intel/llvm/pull/4975

Unfortunately the PR for the tests got merged first and the actual fix stayed in review, just re-requested review for it today.

Reply:
I'll revert these changes if your PR gets approval